HISTORY.txt

```diff
@@ -5,9 +5,114 @@
 Old Release Notes
 =================
 
-=============
-Release Notes
-=============
+Theano 0.10.0beta3 (20th of September, 2017)
+============================================
+
+This release contains new features, improvements and bug fixes to prepare the upcoming release candidate.
+
+We recommend that every developer updates to this version.
+
+Highlights:
+ - conda packages now available and updated in our own conda channel ``mila-udem``.
+   To install it: ``conda install -c mila-udem/label/pre theano pygpu``
```

Comment:
This should be:

conda install -c mila-udem/label/pre -c mila-udem theano pygpu

to have the latest release of pygpu

Reply:
Fixed